src/otx/core/model/module/base.py

```diff
@@ -156,7 +151,7 @@ def load_state_dict(self, ckpt: dict[str, Any], *args, **kwargs) -> None:
         """
         if is_ckpt_from_otx_v1(ckpt):
             model_state_dict = ckpt["model"]["state_dict"]
-            state_dict = self._load_from_prev_otx_ckpt(model_state_dict)
+            state_dict = self._load_from_otx_v1_ckpt(model_state_dict)
```

Comment:
Can we just call `self.model`'s function and return here?
```suggestion
            state_dict = self.model.load_from_otx_v1_ckpt(model_state_dict)
            return
```

Reply:
Changed.